allennlp/tests/fixtures/tools/py2md/basic_example.py

```diff
@@ -0,0 +1,111 @@
+"""
+This is a docstring.
+"""
+
+SOME_GLOBAL_VAR = "Ahhhh I'm a global var!!"
+"""
+This is a global var.
+"""
+
+
+def func_with_no_args():
+    """
+    This function has no args.
+    """
+    return None
+
+
+def func_with_args(a: int, b: int, c: int = 3) -> int:
+    """
+    This function has some args.
+
+    # Parameters
+
+    a : `int`
+        A number.
+    b : `int`
+        Another number.
+    c : `int`, optional (default = `3`)
+        Yet another number.
+
+    # Returns
+
+    `int`
+        The result of `a + b * c`.
+    """
+    return a + b * c
+
+
+class SomeClass:
+    """
+    I'm a class!
+
+    # Attributes
+
+    x : `float`
+        This attribute is called `x`.
+    """
+
```

Comment:
What about class variables and their docstrings? These aren't handled well in our current docs, I think.

Reply:
So, I *thought* that was handled, but I just dug in a little more and realized there's a bug where class-level attributes don't render when they have type-hints.

I just submitted an issue to pydoc-markdown.

https://github.com/NiklasRosenstein/pydoc-markdown/issues/115

Reply:
This has been fixed